src/platforms/ctxlink/README.md

```diff
@@ -0,0 +1,49 @@
+# Firmware BMP for ctxLink Wireless Probe
+
+Allows the use of ctxLink board as a Black Magic Probe.
+
+## LED assignments
+ * PB2 -> LED0 -> Green - Running LED
+ * PC6 -> LED1 -> Orange - Idle LED
+ * PC8 -> LED2 -> Red - Error
+ * PC9 -> LED3 -> Green - ctxLink Mode
+
+## Connections
+
+* PA3: TDI
+* PA4: TMS/SWDIO
+* PA5: TCK/SWCLK
+* PC7: TDO/TRACESWO
+* PA2: TRST
+* PA2: nRST
+
+## How to Flash with DFU
+
+After build:
+
+1) `apt install dfu-util`
+2) Force ctxLink into system bootloader mode by holding down the Mode switch while
+   pressing reset. Release reset followed by Mode. System bootloader should appear.
+3) `dfu-util -a 0 --dfuse-address 0x08000000 -D blackmagic_ctxlink_firmware.elf`
```

Comment:
`dfu-util` needs the .bin here as it throws the contents of w/e file given verbatim at the device.